src/services/callExtAPI.ts

```diff
@@ -0,0 +1,18 @@
+import axios, {AxiosResponse} from "axios";
+
+export async function callExtAPI<T>(url:string): Promise<AxiosResponse> {
```

Comment:
Change to class.

Reply:
Done (functionality moved to Controllers)